src/I18n/languages/cz.json

```diff
@@ -10,6 +10,7 @@
       "restoreTenantsConfirm":"Chcete opravdu obnovit původní seznam organizací?",
```

Comment:
Should have the same line numbers than the other i18n files.

Reply:
Yes, I will do a dedicated PR and use Remi's script to align this file